net.solarnetwork.node.weather.ibm.wc/src/net/solarnetwork/node/weather/ibm/wc/DatumPeriod.java

```diff
@@ -0,0 +1,6 @@
+
+package net.solarnetwork.node.weather.ibm.wc;
+
+public interface DatumPeriod {
+
```

Comment:
The usefulness of this interface seems questionable, unless you added something like

```java
String getPeriod();
```

and then used that method... but even still I wonder if you think this interface is necessary at all?